.github/workflows/stale.yml

```diff
@@ -0,0 +1,34 @@
+# Copyright The Notary Project Authors.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: "Close stale issues and PRs"
```

Comment:
Can we reuse this workflow across repos ?

Reply:
@priteshbandi Do you mean using one workflow across repos? If yes, I do not know how to do it. Actually it is the same template used across repos with some small differences, for example, some descriptions are a bit different for each repository.